Created a new article (#17100)
PostHog/posthog.com: contents/blog/collaborating-with-product-teams-as-a-pmm.md

---

> title: Collaborating with product teams (as a PMM)

Reviewer: Maybe you could have more fun with the title, maybe something like 'A marketer's guide to building a kickass partnership with product teams'?

date: 2026-05-28
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
    - sara-miteva
featuredImage: >-
    https://res.cloudinary.com/dmukukwp6/image/upload/Template_cover_15_8cf985674a.jpg
featuredImageType: full
category: Inside PostHog
tags:
    - Marketing
---

> Before [Cleo](/community/profiles/36864) and I joined as Product Marketers (PMMs), PostHog only had our [Marketing Lead, Joe](/community/profiles/29070), who was doing PMM work with a bunch of other stuff at the same time. This was the first time teams got dedicated PMMs, and also the first time some of them got to work with a PMM, so we had to find the best way to collaborate with them while respecting [our company culture](/handbook/company/culture).

Reviewer 2: ```suggestion
Before [Cleo](/community/profiles/36864) and I joined as Product Marketers (PMMs), PostHog only had [Joe](/community/profiles/29070), who was doing PMM work and a bunch of other stuff at the same time. This was the first time teams got dedicated PMMs, and also the first time some of them got to work with a PMM, so we had to find the best way to collaborate with them while respecting [our company culture](/handbook/company/culture).
```

> Before [Cleo](/community/profiles/36864) and I joined as Product Marketers (PMMs), PostHog only had our [Marketing Lead, Joe](/community/profiles/29070), who was doing PMM work with a bunch of other stuff at the same time. This was the first time teams got dedicated PMMs, and also the first time some of them got to work with a PMM, so we had to find the best way to collaborate with them while respecting [our company culture](/handbook/company/culture).

Reviewer: ```suggestion
Before [Cleo](/community/profiles/36864) and I joined as Product Marketers (PMMs), PostHog only had our [Marketing Lead, Joe](/community/profiles/29070), who was doing PMM work with a bunch of other stuff at the same time. This was the first time teams got dedicated PMMs, and also the first time some of them got to work with a PMM full stop. We had to find the best way to collaborate with them while respecting [our company culture](/handbook/company/culture).
```

One of the first things you learn when joining PostHog is that [collaboration sucks](https://newsletter.posthog.com/p/collaboration-sucks). You're the driver and you need to ship your own stuff. Collaborate only when necessary.

![collaboration sucks](https://res.cloudinary.com/dmukukwp6/image/upload/stop_collaborating_7550066220.webp)

> Well, the relationship between Product Managers (PMs) and PMMs is one of those necessary situations when you *need to* collaborate to successfully launch and grow your products. Knowing that collaboration with PMs is one of the biggest pains for a PMM, I wanted to share some collaboration best practices from our experience so far.

Reviewer 2: ```suggestion
Well, the relationship between Product Managers (PMs) and PMMs is one of those necessary situations when you *need to* collaborate to successfully launch and grow your products. I wanted to share some ways I've learned to do this at PostHog.
```

I think this starts to feel stronger when it's your personal experience and opinionated take. Less "Here are the best practices we have learned", more "Here's what I think works."

You are the one who is acing this internally, so I think you should just be really bold and clear about your thoughts -- even if it would push you to being an iconoclast.

## Initiate the first contact

As a PMM joining a product team that, in most cases, has already been established, you have the difficult task of making yourself visible and proving you came here for a reason. Most engineers, and even PMs, don't get to work with marketing people often, so they don't even know what your job is.

> That's why I believe you, the PMM, should initiate the first contact. Book a call with your PM (even if you're going to continue to work with them async, the first intro should be a call), tell them what your job is, and ask for more context around the product. Here are some things worth covering in this first meeting:

Reviewer 2: Similar to above, really. The content here is great, but I think we can tell it more powerfully by grounding it in your personal opinion and experience. 

```suggestion
When I joined PostHog the team I was supporting was already established, but most of them had never worked with a product marketer before. Many didn't know what one does.

I decided I had to make myself visible and prove I could be valuable. So, I made first contact by having a call with the PM, Abe, rather than waiting for them to come to me. 

Here's what we talked about:
```

- What is the current revenue of the product, and what is the revenue goal (for the quarter, year, etc.)?
- What is a bigger gap - activation or retention?
- Do you know if there were any previous marketing efforts around this product?
- Where do most users come from?
- Do you see any obvious marketing opportunities that I should focus on first?

Ask for the customer interview bank, any competitive research they might have, or any other resources that could be helpful for your particular product.

> Since async is the default way of working in PostHog, I was very careful with scheduling meetings at first. However, the teams got very excited when I announced that I would be supporting their products, and were very eager to meet and give me as much information as they could.

Reviewer 2: <3

![collaborating with PMs](https://res.cloudinary.com/dmukukwp6/image/upload/PM_COLLAB_9b73b35feb.png)

## Live in your team's channel

If you're using Slack or any similar tool, your team probably has a dedicated channel. This is the most important place you need to check every day, or at least every Monday.

> The PMs and the engineers might not always know which feature is marketable. You should follow the threads and figure it out yourself. Jump in wherever you think there's an opportunity, and the team will most likely get excited that you want to show users what they've been working on.

Reviewer 2: This is a great point around pull/push tactics. Don't wait for them to pull you in. Push ideas into the team and see how they're received.

## Spend time in person & in meetings

> PostHog is fully remote, so you usually only meet teammates a couple of times a year. If this is the case with you, try to make the most of those moments. This year we went to Barbados, and I booked some dedicated time with both of my PMs to do a growth review and discuss the parts of the product we want to focus on next.

Reviewer 2: I think this section should be built out more, as well as grounded in your experience. We should explain what a growth review is, what you bring to them, and how you prepare. 

Talking about Barbados is great. If we're grounding this in your experience we could bring a story out of this: what was something helpful you got?

> In addition to the in-person gatherings, you should regularly join your team's sprint planning (or any similar meetings). Even if they are too technical, you should show up and be present. The team should see that you're interested in what they're working on and hear you asking questions. If you feel bad for not contributing, you can always give them a short overview of what you're working on related to their product.

Reviewer: A point I would make here based on my own experience so feel free to take it or leave it - Very few PMMs are going to have a full technical understanding of the product when they join a company, joining these meetings is a good way to ask questions and learn more about the product

## Loop your PM in on your marketing activities

> At PostHog, most of us create an individual [quarterly planning issue](https://github.com/issues/assigned?issue=PostHog%7Crequests-for-comments-public%7C524). I regularly send this issue to my PMs to get their thoughts on the things I'm planning to do in the upcoming quarter, related to their product.

Reviewer: Could also add that this is a good way to learn about gaps in your visibility of whats launching that quarter

Also, I regularly send them my campaign plans, creative directions, blogs, etc. The core assets are non-negotiable - the PM should always take a look at the product page, launch email, and launch blog.

> Even if they don't have marketing feedback, they should be in the loop and they should get excited about the way you're promoting their product. Also, sometimes they have competitive research or data that will help you create your assets, and you'll only know this if you share what you're working on and get feedback.

Reviewer 2: I think "in the loop" reads a _little_ B2B. Maybe we can make it clear that this isn't performative collaboration by linking to specific launch plans on GitHub and highlighting places where Abe and you have levelled things up by building on each others' work?

Getting practical here is important, I think. The message shouldn't be "It's important to tell people what you're doing" -- it should be "I tell them what I'm doing and this is how it helps".

> Even if they don't have marketing feedback, they should be in the loop and they should get excited about the way you're promoting their product. Also, sometimes they have competitive research or data that will help you create your assets, and you'll only know this if you share what you're working on and get feedback.

Reviewer: I'd also add that this is a way to double check the technical accuracy of your copy and make sure you and the PM have the same picture of the product from a feature perspective as well as who you are trying to market this too - when Anna reviews my work she gives feedback based on her knowledge of the audience and whether they will or won't respond to certain phrases too!

## Monthly growth review

> Recently, we started doing monthly 1:1 meetings with PMs where we go through the product growth for the last month and try to identify the biggest gaps. For example, for one of the products I was working on, we realized that retention is low, so we decided to start sending monthly updates to users that include everything the team has been working on and what's coming next.

Reviewer 2: Again, let's just be specific. Logs retention is low. I found that out via the PM. I proposed trying this new thing. He hadn't thought of that. This is why growth reviews are important for both PMs and PMMs.

## Don't allow yourself to become an afterthought

> Yes, you're the one joining an already established team, but you should also express your preferred ways of working. If you notice you're getting included in conversations too late, react. This happened to me when I got pulled in to work on some integration launches. Once engineers started including me early, I had more context, and as a result, we were able to do much more before launch day.

Reviewer: Maybe not something you do, but showing the impact of the marketing in a sort of campaign retro to the PM and team lead could be a good way to help them realise what impact you can have with more time?
